app/client/src/layout/Page/Page.tsx

```diff
@@ -11,6 +11,7 @@ const useStyles = makeStyles({
         flexDirection: 'column',
         flex: '1 1 100%',
         overflowY: 'auto',
+        overflowX: 'hidden',
```

Comment:
Out of curiosity, what behavior is being fixed here?

Reply:
I encountered a horizontal scrollbar when making the landing page which may be due to the absolute elements, so I made the horizontal overflow hidden to fix this.

Reply:
Hmm, when breaking up the landing page is done, lets revisit this.  I'm a bit hesitant to modify this file since it's used on every single other page. Unfortunately, we don't have proper testing in place just yet to make sure that nothing else unintentionally broke.  There may be another solution that is good enough and doesn't risk breaking anything else.